vtex/utils/transform.ts

```diff
@@ -494,16 +494,21 @@ const legacyToProductGroupAdditionalProperties = (product: LegacyProductVTEX) =>
     );
   }) ?? [];
 
-const toProductGroupAdditionalProperties = ({ properties = [] }: ProductVTEX) =>
-  properties.flatMap(({ name, values }) =>
-    values.map(
-      (value) =>
-        ({
-          "@type": "PropertyValue",
-          name,
-          value,
-          valueReference: "PROPERTY" as string,
-        }) as const,
+const toProductGroupAdditionalProperties = (
+  { specificationGroups = [] }: ProductVTEX,
+) =>
+  specificationGroups.flatMap(({ name: groupName, specifications }) =>
+    specifications.flatMap(({ name, values }) =>
+      values.map(
+        (value) =>
+          ({
+            "@type": "PropertyValue",
+            name,
+            value,
+            propertyID: groupName,
```

Comment:
Does this property exists in PropertyValue?

Reply:
yes